### ISSUE TITLE
[0.2.0-rc7] Bad Address

### ISSUE THREAD

**Issue by halkeye** [Place1/wg-access-server#44]
My config has:
```
    vpn:
      rules:
        allowedNetworks:
          - 0.0.0.0/0
          - 128.0.0.0/0
          - ::/0
          - 172.16.10.0/24
```

And it seems to generate a config
```
AllowedIPs = 10.44.0.1/32, 10.44.0.0/24, 10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16, 200.0.0.0/5, 172.64.0.0/10, 172.128.0.0/9, 12.0.0.0/6, 16.0.0.0/4, 11.0.0.0/8, 32.0.0.0/3, 128.0.0.0/3, 196.0.0.0/6, 64.0.0.0/2, 172.0.0.0/12, 194.0.0.0/7, 192.160.0.0/13, 192.0.0.0/9, 192.170.0.0/15, 160.0.0.0/5, 192.128.0.0/11, 193.0.0.0/8, 208.0.0.0/4, 192.172.0.0/14, 176.0.0.0/4, 192.169.0.0/16, 0.0.0.0/5, 174.0.0.0/7, 192.176.0.0/12, 192.192.0.0/10, 8.0.0.0/7, 172.32.0.0/11, 173.0.0.0/8, 168.0.0.0/6, 0.0.0.0/0, 128.0.0.0/0, ::/0, 172.16.10.0/24
```

Which wireguard for android goes "Error bringing up tunnel: Bad Address"

I havn't tracked it down to the actual cause though

**Comment by halkeye** [comment]
Changing it to just "- 0.0.0.0/0" does work though, but it is a breaking change

**Comment by verdin** [comment]
It might be this network
`- 128.0.0.0/0`
 that's not a valid cidr it should likely be 
`- 128.0.0.0/8 `

When running Wireguard on linux with that cider it get this error 
`Warning: AllowedIP has nonzero host part: 128.0.0.0/0`

**Comment by Place1** [comment]
I think @verdin may be correct but I've still introduced buts in this RC.

The reason for the large list of allows IPs being added is because I added a new set of config values for VpnLan/ServerLan/Internet which default to true.

I've encountered a different issue with the above where "allowInternet" adds "64.0.0.0/2" to the client's allowedIPs. Turns out that this breaks some clients like iOS with a route loop issue ([read more](https://wiki.archlinux.org/index.php/WireGuard#Loop_routing)) because, for me, that CIDR matched the VPN endpoint itself.

It's a little surprising that this happened because the iOS & mac client apps have a "Exclude Private IPs" checkbox that adds these same CIDRs to allowed IPs. The config works for me on Windows but ios/mac break. Weird. If anyone knows something about this i'd love to know what's up with this :P

I think i'll have to change the feature that I introduced in this RC to fix the above issue and probably the one you've encountered @halkeye.

Instead i'll simply have allowedIPs as a config value with a default of "0.0.0.0/1, 128.0.0.0/1"
